app/views/catalog/_thumbnail_list_collection.html.erb

```diff
@@ -1,3 +1,9 @@
 <div class="col-md-2">
-  <span class="<%= Hyrax::ModelIcon.css_class_for(Collection) %> collection-icon-search" aria-hidden="true"></span>
+  <% collection_presenter = Hyrax::CollectionPresenter.new(document, current_ability) %>
+  <% if (collection_presenter.thumbnail_path == nil) %>
+    <span class="<%= Hyrax::ModelIcon.css_class_for(Collection) %> collection-icon-search" aria-hidden="true"></span>
```

Comment:
Is this line still tested or is it worth testing this line (maybe in a view test instead of a feature test)?

Reply:
Thanks for the suggestions. I've added tests for it. Please review the updated commits.

Reply:
I'm seeing the test for the new case displaying the thumbnail (line 18 of the view spec) but nothing for the original case that displays the icon.  Does the feature test cover this case or is the feature test finding the thumbnail as well?  I think it would be nice to retain the original test in the view spec making for two tests that cover both cases.